### PR TITLE
welcome_run-d-program-locally

### PR DESCRIPTION
**Pull request by vladdeSV**

### CODE REVIEW

**Comment by vladdeSV** [comment]
Ping @wilzbach

**Comment by wilzbach** [comment]
> Ping @wilzbach

Did someone already give the go for this PR? (originally: https://github.com/stonemaster/dlang-tour/pull/438)

@nordlow @Vild feel free to press merge if so ;-)

> changed the title from Add running page to [swedish] sv_welcome_run-d-program-locally

Oh that's no longer necessary - this repo is entirely yours!

**Comment by vladdeSV** [comment]
@wilzbach Should we have some sort of syntax for how the branches are laid out?

I'd suggest at least something like `welcome_run-d-program-locally`. It is clear what part and which page is being added

**Comment by wilzbach** [comment]
> @wilzbach Should we have some sort of syntax for how the branches are laid out?

Oh I was just talking about the "[swedish]" prefix.

> I'd suggest at least something like welcome_run-d-program-locally. It is clear what part and which page is being added

That sounds very reasonable to me :+1:

**Comment by wilzbach** [comment]
btw that might be motivating: the first bits are already part of the Tour and online:

https://tour.dlang.org/tour/sv/welcome/welcome-to-d

**Comment by vladdeSV** [comment]
Updated the index to include this file, and also change the title to `Välkommen`

**Comment by wilzbach** [comment]
> Updated the index to include this file, and also change the title to Välkommen

While not related to this PR, you might also want to update the overall title which is currently "The Dlang Tour"

https://github.com/dlang-tour/swedish/blob/master/index.yml (line 1)

(FYI: the other control texts aren't multi-lingual yet, but will be so soon)

File: welcome/run-d-program-locally.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+    dub init hello
+
+Exekvering av kommandot `dub` i denna mapp kommer hämta alla beroenden, kompilera applikationen
+och köra den.
+`dub build` kommer bygga (kompilera och länka) projektet.
+
+Granska [online dokumentationen](https://code.dlang.org/docs/commandline) eller kör `dub help` för att få en överblick av tillgängliga kommandon och funktioner.
+
+Alla tillgängliga DUB-paket kan granskas via dubs [webbgränssnitt](https://code.dlang.org).
```
Comment (jacob-carlborg) [inline welcome/run-d-program-locally.md]: Inconsistent spelling, `DUB` and `dubs`.

File: welcome/run-d-program-locally.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+På UNIX system kommer 'shebang'-raden `#!/usr/bin/env rdmd` på första raden av en D-källkodsfil att tillåta
+skript liknande användning.
+
+Granska [online dokumentationen](https://dlang.org/rdmd.html) eller kör `rdmd --help` för att få en överblick av tillgängliga flaggor.
+
+### Pakethanteraren `dub`
+
+
+Ds officiella pakethanterare är [`dub`](http://code.dlang.org). När `dub` är installerat
+lokalt, kan ett nytt projekt, exempelvis kallat `hello`, skapas med kommandot:
```
Comment (jacob-carlborg) [inline welcome/run-d-program-locally.md]: I don't `lokalt` is necessary.

File: welcome/run-d-program-locally.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
+# Köra D program lokalt
```
Comment (jacob-carlborg) [inline welcome/run-d-program-locally.md]: I don't think `lokalt` is necessary.

**Comment by vladdeSV** [comment]
@jacob-carlborg Thank you for your suggestions. Slam that merge button if you feel it is finished :+1:

**Comment by jacob-carlborg** [comment]
Looking good, please squash commits.

**Comment by wilzbach** [comment]
> please squash commits.

Github can do this automatically, just press on merge and select the squash option ;-)

**Comment by jacob-carlborg** [comment]
Forgot about that feature, done. Thanks.

**Comment by wilzbach** [comment]
> Forgot about that feature, done. Thanks.

You are very welcome.  It is quite convenient and a shame many repositories don't use it (yet), because it's still so "new" :/

**Comment by jacob-carlborg** [comment]
Hmm, I do not like how the history turned out after squashing the commits on GitHub. It used the branch name as the first part of the commit message, not the actual text from the text area as I expected 😠.

**Comment by wilzbach** [comment]
> It used the branch name as the first part of the commit message, not the actual text from the text area as I expected 😠.

Normally one is able to change/adapt this directly in the squash interface, maybe you missed it? 

> Hmm, I do not like how the history turned out after squashing the commits on GitHub. 

It could be worse: 
https://github.com/dlang/phobos/pull/4662

**Comment by jacob-carlborg** [comment]
> Normally one is able to change/adapt this directly in the squash interface, maybe you missed it?

I did that, and I change the commit message. I did it on my phone so I could have missed some part of the commit message, but I don't think I did.

> It could be worse

Yeah, I've seen a Phobos PR being merged with around 100 commits, most of the fix simple things like spell errors.

**Pull request merged**